Src/MoneyFox.Win/MoneyFox.Win/Converter/PaymentTypeToIntConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace MoneyFox.Win.Converter;
+
+using Core.Aggregates.Payments;
+using Core.Resources;
+using Microsoft.UI.Xaml.Data;
+using System;
+
+public class PaymentTypeToIntConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, string language)
```

Comment:
This is also easy testable with a unit Test. The Convert Back Method for me doens't have to have one, when we don't use that method. But the convert method itself would be nice to be covered.

Reply:
I deleted this class when I switched from using int to enum for the Payment Type filter. However, it was replaced with a new class called PaymentTypeFilterConverter. All of the Converter classes are in the MoneyFox.Win project which the MoneyFox.Core.Tests project does not reference. I can add a reference to the project and create a unit test if you think it's valuable, but this would suggest that none of the unit tests cover functionality in MoneyFox.Win. Am I missing something?

Reply:
aah, shoot. you are right. for the win project we currently do not have a test project, since with UWP it didn't run on the CI/CD pipeline. Actually have to check again with WinUI.